arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import brace from 'brace';
+import AceEditor from 'react-ace';
+
+import 'brace/theme/xcode';
+
+import AqlMode from './AqlMode'
+
+import {
+
+} from '../../actions/document'
```

Comment:
This looks like it doesn't import anything. Is this intentional?

Reply:
There will be some action functions in the future.